content/en/docs/unit-testing.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "Unit Testing with the AWS SDK for Go V2"
+linkTitle: "Testing"
+description: "How to mock the AWS SDK for Go V2 when unit testing your application."
+---
+
+You can mock out the AWS SDK for Go V2 when unit testing your application by
+using Go interfaces. Using interface definitions you define the set of
+operations required by your application, and provide mock implementations of
+this interface when unit testing. You can follow this pattern to unit testing
+service client operations, paginators, and waiters.
+
+## Mocking Client Operations
+
+In this example, `S3GetObjectAPI` is an interface that defines the set of
+{{% alias service=S3 %}} API operations required by the `GetObjectFromS3`
+function. `s3GetObjectAPI` is satisfied by the {{% alias service=S3 %}}
+client's [GetObject]({{< apiref "service/s3#Client.GetObject" >}}) method.
+
+```go
+import "context"
+import "github.com/aws/aws-sdk-go-v2/service/s3"
+
+// ...
+
+type S3GetObjectAPI interface {
+	PutObject(ctx context.Context, params *s3.GetObjectInput, optFns ...func(*s3.Options)) (*s3.GetObjectOutput, error)
```

Comment:
Is this intentional? confusing that GetObject is named as PutObject.

Reply:
Not intentional